tests/cli/test_lib.py

```diff
@@ -55,6 +56,20 @@ def ReadOutput(self):
 
     return output_data
 
+  # TODO: refactor this method https://github.com/log2timeline/plaso/issues/1963
```

Comment:
On a second pass the intent of this class is to write output including encoding seeing some output format might specify a specific encoding. So that output is consistent cross-platform. Please undo these changes, seeing the no longer reflect what needs to be tested.

Reply:
Per conversation, I've changed the TestOutputWriter to just deal with strings, with no conversion between outputs and test fixtures (expected output).